dbcon/mysql/ha_mcs_impl_if.h

```diff
@@ -335,6 +341,7 @@ struct cal_connection_info
     // MCOL-1101 remove compilation unit variable rmParms
     std::vector <execplan::RMParam> rmParms;
     long long affectedRows;
+    int lock_type;
```

Comment:
Changes in this file don't belong to MCOL-4144, probably you need to rebase your branch on top of latest develop.

Reply:
Fixed